playground/nested-deps/__tests__/nested-deps.spec.ts

```diff
@@ -9,5 +9,7 @@ test('handle nested package', async () => {
   expect(await page.textContent('.side-c')).toBe(c)
   expect(await page.textContent('.d')).toBe('D@1.0.0')
   expect(await page.textContent('.nested-d')).toBe('D-nested@1.0.0')
-  expect(await page.textContent('.nested-e')).toBe('1')
+
+  // TODO: Review if the test is correct
+  // expect(await page.textContent('.nested-e')).toBe('1')
```

Comment:
I think you're right. This might have not been actually fixed in https://github.com/vitejs/vite/pull/5411 at all. There's https://github.com/vitejs/vite/issues/5688 that shows this is still a issue. We can further investigate this if this is merged.

Reply:
So we can merge with this test skipped then